scrapypuppeteer/middleware.py

```diff
@@ -75,7 +65,7 @@ class PuppeteerServiceDownloaderMiddleware:
     SERVICE_META_SETTING = "PUPPETEER_INCLUDE_META"
     DEFAULT_INCLUDE_HEADERS = ["Cookie"]  # TODO send them separately
 
-    PUPPETEER_LOCAL_SETTING = "PUPPETEER_LOCAL"
+    EXECUTION_METHOD_SETTING = "EXECUTION_METHOD"
```

Comment:
Since you change the name of the setting, mark it in `Type of change` (look at PR template).